db/migrate/20230120165646_create_post.rb

```diff
@@ -1,12 +1,14 @@
 class CreatePost < ActiveRecord::Migration[7.0]
-  def change
-    create_table :posts do |t|
-      t.string :Title
-      t.string :Text
-      t.integer :CommentsCounter
-      t.integer :LikesCounter
+   def change
+      create_table :posts do |t|
+         t.string :Title
+         t.string :Text
+         t.datetime :CreatedAt
+         t.datetime :UpdatedAt
```

Comment:
- [ ] `t.timestams` will crate updated_at and created_at columns, which means that you don't need to create extra columns, please 🙏🏽  remove UpdateAt and CreateAt columns to avoid future conflicts.